evals/registry/evals/test-modelgraded-battle.yaml

```diff
@@ -9,6 +9,7 @@ joke-animals-vs-fruits.dev.v0:
     samples_jsonl: test_multiio/battles/joke_animals_vs_fruits.jsonl
     eval_type: cot_classify
     modelgraded_spec: battle
+    eval_completion_fn: gpt-3.5-turbo
```

Comment:
nit: This isn't needed since it's already the default?

Reply:
I added it to demonstrate how this would be done in practice